src/docdb/tree/DocDBDatabaseTreeItemBase.ts

```diff
@@ -109,21 +94,23 @@ export abstract class DocDBDatabaseTreeItemBase extends DocDBTreeItemBase<Collec
             validateInput: (input: string) => validateThroughput(isFixed, input)
         }));
 
-        const options = { offerThroughput: throughput };
+        let partitionKey: PartitionKeyDefinition | undefined;
+        if (!isFixed) {
+            if (!partitionKeyInput.startsWith('/')) {
+                partitionKeyInput = '/' + partitionKeyInput;
+            }
+            partitionKey = { paths: [partitionKeyInput] };
```

Comment:
We don't need to specify "kind" anymore?

Reply:
`kind` as a property for partitionKey is not available with the new SDK and haven't found its usage anywhere in samples. I am trying to find documentation to confirm this.